packages/components/src/styles/list/list.stories.ts

```diff
@@ -1,26 +1,20 @@
 import '../../solid-components';
-
+import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
 
 const { argTypes, parameters } = storybookDefaults('sd-list');
 const { overrideArgs } = storybookHelpers('sd-list');
 const { generateTemplate } = storybookTemplate('sd-list');
 
-// HTML helper to get syntax highlighting and formatting in the template string
-const html = String.raw;
-
 /**
- * Use lists to make blocks of text easier to read and to divide information into manageable sections. <br>
+ * Used to make blocks of text easier to read and to divide information into manageable sections.
+ *
  * Text lists can be numbered, have bullet points, or be supplemented by content symbols. Text can be bolded or linked.
- * <br>
- * <b>Variants</b><br>
- * <li>Unordered list group is used when there is no specific sequence or order to the items.</li>
- * <li>Ordered list group is used when the items have a specific sequence or count.</li>
- * <li>Icon list group is used when more illustration is needed.</li>
```

Comment:
* <li>Icon list group: used when a content icon is needed to illustrate the text content.</li>

Reply:
Updated